test/remote/gateways/remote_adyen_test.rb

```diff
@@ -17,6 +17,16 @@ def setup
 
     @declined_card = credit_card('4000300011112220')
 
+    @maestro = credit_card(
+      '5500000000000004',
+      month: 8,
+      year: 2018,
+      first_name: 'John',
+      last_name: 'Smith',
+      verification_value: '737',
+      brand: 'mastercard'
```

Comment:
Isn't maestro is a legit brand in AM?

Reply:
It is, but this feature is explicitly testing the ability to _override_ the brand via those fields for use in Core; this is intentional. Ping me on Slack and I can link you to the Zendesk ticket for more background.

Reply:
Although this variable should be changed to `@mastercard`; let me do that.

Reply:
I see. Perhaps the commit/changelog wording could be more accurate re:overriding?